hypershift-operator/metrics.go

```diff
@@ -67,6 +75,11 @@ func newMetrics(client crclient.Client, log logr.Logger) *hypershiftMetrics {
 			Help: "Time in seconds it took from initial cluster creation to HostedClusterAvailable condition becoming true",
 			Name: "hypershift_cluster_available_duration_seconds",
 		}, []string{"name"}),
+		// hostedClusterInfo is a metric that contains HostedCluster object info.
+		hostedClusterInfo: prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
@enxebre I would expect to produce individual metrics for the different pieces of information as in
https://github.com/openshift/osd-metrics-exporter/blob/main/pkg/metrics/aggregator.go
Having all these labels in a single metric will result in a high cardinality metric that is a permutation of all possible label values.

Reply:
+1

Reply:
+1